eng/doc/fips/README.md

```diff
@@ -435,6 +435,10 @@ This list of major changes is intended for quick reference and for access to his
 
 ### Go 1.25 (Aug 2025)
 
+- Running `go version -m` on a binary which uses a system crypto backend now shows the `microsoft_systemcrypto=1` build setting.
+
+ - It is now possible to build a binary that doesn't depend on a crypto package using invalid configurations, for example `GOOS=linux CGO_ENABLED=0 GOEXPERIMENT=systemcrypto`.
```

Comment:
The doc should also note this behavior in the `Build option to use Go crypto if the backend compatibility check fails` section: acknowledge that when someone sees this error, it might not be because of their build setup, but rather because their source code might not have used `crypto` before. Another possible fix is to remove `crypto`.

Reply:
Good call. The fix you mention seems unrealistic, though, so I won't add it 😛.

Reply:
I get your point, but there are very few cases where not using crypto is the right fix, and I don't want to encourage people implementing their bespoke crypto algorithms(or third-party devs) that don't depend on crypto just to make the compiler happy.

Reply:
Done!

Reply:
> I don't want to encourage people implementing their bespoke crypto algorithms(or third-party deps) that don't depend on crypto just to make the compiler happy.

That's a good point, hopefully someone on their project will understand the crypto policy and make sure this doesn't happen, but the way the policy and enforcement works will put people in a difficult situation and I'm sure *someone* will take our doc as an inspiration for a bad workaround.

In the past we've taken an openness approach in this doc (describe how things work, even internal workarounds, because motivated people will find it anyway and just waste their time), but that was a while ago. Perhaps also something to reevaluate on a broader scale.